### ISSUE TITLE
Issues when using .slider class for slider handle

### ISSUE THREAD

**Issue by PeterTheOne** [seiyria/bootstrap-slider#445]
I was using ``.slider`` class for my slider handle element like this ``<div class="slider"></div>`` and then I had problems when calling ``$('.slider').slider('setValue', 5);``. I got a ``cannot call methods on slider prior to initialization`` error, this is because one of the elements bootstrap-slider creates has a ``.slider`` class.

It would be nice to have a warning about this in the readme.md or to change the class name to something less common. ^^

**Comment by seiyria** [comment]
I think it's easier for you to add `.slider-container` (instead of `.slider` -- it's more explicit as to what it does) or hold references separately, sorry. The error is there to let you know something's going wrong.

**Comment by PeterTheOne** [comment]
yes, you're right, I shouldn't have called it ``.slider``, but I think it's a common mistake to make. The error itself is not a problem, a note about this in the readme would be nice never the less.

**Comment by dionoid** [comment]
I had exactly the same issue by giving my slider container-div's the "slider" class. Would be nice if the internal slider classes are less obvious to override.

**Comment by PeterTheOne** [comment]
Or maybe use some kind of prefix for the slider class name.

**Comment by rovolution** [comment]
Would some sort of console warning be good to indicate to the developer that the classname they selected for the slider container is invalid?

**Comment by PeterTheOne** [comment]
@rovolution  Console warning would be a big improvement! Maybe with reference to the doc. (Can we reopen this issue ^^)
